controllers/oauth.js

```diff
@@ -35,16 +35,20 @@ const FxAOAuthClient = new ClientOAuth2({
   scopes: ["profile"],
 });
 
+
 function init(req, res, next, client = FxAOAuthClient) {
   // Set a random state string in a cookie so that we can verify
   // the user when they're redirected back to us after auth.
   const state = crypto.randomBytes(40).toString("hex");
   req.session.state = state;
   const url = new URL(client.code.getUri({state}));
+  const fxaParams = new URL(req.url, AppConstants.SERVER_URL);
+
   url.searchParams.append("access_type", "offline");
   url.searchParams.append("action", "email");
-  if (req.query.scanned) {
-    url.searchParams.append("email", req.query.scanned);
```

Comment:
Hmm ... I thought I saw a bug where this makes the url parameter in the redirect to `stable.dev.lcip.org/oauth/signin?`...`scanned=<email>` ... which breaks the pre-population logic on the FxA page:

![image](https://user-images.githubusercontent.com/71928/54129686-ccd6b500-43dc-11e9-95dc-5b57521bbbf9.png)

But I can't reproduce it. So I won't block this but let's keep it in mind in case we see this bug happen during today's deploy.